debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/testhelper/RestExtensionTestInfrastructure.java

```diff
@@ -155,8 +161,8 @@ public static void setupDebeziumContainer(String connectorVersion, String restEx
         final String debeziumVersion = debeziumContainerImageVersion.startsWith("1.2") ? "1.2.5.Final" : connectorVersion;
         String baseImageName = registry + "debezium/connect-base:" + debeziumContainerImageVersion;
         DEBEZIUM_CONTAINER = new DebeziumContainer(new ImageFromDockerfile("debezium/connect-rest-test:" + debeziumVersion)
-                .withFileFromPath(".", Paths.get("target/"))
-                .withFileFromPath("Dockerfile", Paths.get("src/test/resources/Dockerfile.rest.test"))
+                .withFileFromPath(".", Paths.get(System.getProperty("user.dir") + "/target/"))
+                .withFileFromPath("Dockerfile", Paths.get(System.getProperty("user.dir") + "/src/test/resources/Dockerfile.rest.test"))
```

Comment:
IIRC `user.dir` assumes the directory from which the process starts, which may not necessarily be what you expect.  Would it be better to have maven (if it isn't already passed) to provide the project directory and we use that instead so that regardless of wherever the build is started from we have the right directory reference to those resources?

Reply:
Yes, but: The code before was working for all connectors but not for the Oracle connector, I would like to get back the previous code that was supposed to work relatively to the module's base dir. But it was not working for the Oracle connector. Ideas?

Reply:
Nope, there is absolutely nothing in the `pom.xml` that I see that would influence or change that behavior that is any different from other connectors.

Reply:
I am using `project.basedir` and `project.build.directory` now. Fixed.